translations/Hindi/special-terms.md

```diff
@@ -0,0 +1,107 @@
+## Terms (शब्द)
+* Message : संदेश , Message in this case is संदेश. There are other words like 
+समाचार and सूचना, but they mean news and information respectively. Besides, 
+संदेश is used in other contexts as well, and in all those contexts it means
+ message. , <user> को संदेश भेजें
+* Private Message (PM) : निजी संदेश (पीएम) , निजी is the first translation we 
+get while translating private. The other translations available (with literal
+ meaning in brackets) are: गैर-सरकारी (un-official, not the right word for 
+ zulip usage) ; व्यक्तिगत (personal, but the idea of private messages isn't 
+ about sharing personal stuff). , <user> को निजी संदेश भेजें
+* Group PM : समूह निजी संदेश , समूह means group, but one meaning also given 
+is team. Therefore group pm can be understood as a way to communicate with 
+only the team members. , <Group name> में एक समूह निजी संदेश भेजें
+* Realm : क्षेत्र , Realm translates to क्षेत्र but re-translating back to 
+English it becomes *area* or *regoin*. This means that switching between
+ realms is lke talking to different people in different areas, which is 
+ how reals work in Zulip. , किसी अन्य क्षेत्र में जाएं
+* Stream : स्ट्रीम , Transliteration is used here as there is no way we can 

```

Comment:
*Dhara* seems like a good option in my opinion to transliteration.

Reply:
धारा cannot be used because like I mentioned, it means a stream of flowing water. So if we use transliteration, it is better as stream is generally understood as a group of subjects, like science stream, etc

Reply:
Haven't changed this due to the above reasons